lib/datadog/tracing/trace_operation.rb

```diff
@@ -299,6 +302,7 @@ def to_digest
           trace_service: service,
           trace_state: @trace_state,
           trace_state_unknown_fields: @trace_state_unknown_fields,
+          trace_remote: (@remote_parent && @active_span.nil?),
```

Comment:
Do we need `@active_span.nil?` here?

Reply:
Yup. If we are creating a digest for an active span then `trace_remote` should be `False`. The digest represent a span that is local to the process.